tools/get_tcwind_csv.py

```diff
@@ -312,6 +321,10 @@ def main():
                         help="Name of the output CSV file, otherwise output to stdout")
     parser.add_argument('--product', required=False, default='DeepCyc',
                         help="Name of the product to query. DeepCyc or Metryc.")
+    parser.add_argument('--deepcyc_version', required=False, default='2.0.7', 
+                        help="DeepCyc version to use.")
+    parser.add_argument('--metryc_version', required=False, default='1.0.5', 
+                        help="Metryc version to use.")
```

Comment:
I think it is better to have 'None' as the default and that will let the API decide on the default version rather than this script.